Source/bmalloc/bmalloc/BPlatform.h

```diff
@@ -332,7 +332,9 @@
 #endif
 
 /* Enable this to put each IsoHeap and other allocation categories into their own malloc heaps, so that tools like vmmap can show how big each heap is. */
+#if !defined(BENABLE_MALLOC_HEAP_BREAKDOWN)
 #define BENABLE_MALLOC_HEAP_BREAKDOWN 0
+#endif
```

Comment:
```suggestion
#if !defined(BENABLE_MALLOC_HEAP_BREAKDOWN)
#if BOS(DARWIN)
#define BENABLE_MALLOC_HEAP_BREAKDOWN 1
#else
#define BENABLE_MALLOC_HEAP_BREAKDOWN 0
#endif
#endif
```

Reply:
that would regress apple ports in the sense that much more code would be compiled

Reply:
OK, I see it's supposed to be off by default.